cmd/entrypoint/subcommands/subcommands_test.go

```diff
@@ -38,6 +38,21 @@ func TestProcessSuccessfulSubcommands(t *testing.T) {
 	if _, ok := returnValue.(SubcommandSuccessful); !ok {
 		t.Errorf("unexpected return value from decode-script command: %v", returnValue)
 	}
+
+	t.Run(StepInitCommand, func(t *testing.T) {
```

Comment:
Nice use of `t.Run` here to segment the test. Possible to reproduce with the two `returnValue` checks above as well to keep them consistent? Something like

```go
t.Run(CopyCommand, func(t *testing.T) {
  // ...
})
t.Run(DecodeScriptCommand, func(t *testing.T) {
  // ...
})
t.Run(StepInitCommand, func(t *testing.T) {
  // ...
})
```

We might be getting into table test territory too.

Reply:
Sure thing! I'll do this in another PR though